Content.Client/GameObjects/Components/Storage/ClientStorageComponent.cs

```diff
@@ -31,7 +32,7 @@ public override void OnAdd()
             base.OnAdd();
 
             Window = new StorageWindow(IoCManager.Resolve<IDisplayManager>())
-            { StorageEntity = this };
+            { StorageEntity = this, Size = new Vector2(180.0f, 320.0f)};
```

Comment:
Again, put this size in the StorageWindow instead.

Reply:
Done